src/components/ConfirmContent.js

```diff
@@ -86,4 +97,4 @@ function ConfirmContent(props) {
 ConfirmContent.propTypes = propTypes;
 ConfirmContent.defaultProps = defaultProps;
 ConfirmContent.displayName = 'ConfirmContent';
-export default withLocalize(ConfirmContent);
+export default compose(withLocalize, withNetwork())(ConfirmContent);
```

Comment:
@samh-nl We can use `useNetwork` hooks instead the HOC. While doing that, we can also replace the current `withLocalize` HOC with the hooks too.